doc/interfaces/supercomputer_data/regions.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Regions"
+filters:
+  - "include-code-files"
+---
+
+
+
+## Regions
+
+Any region that can be selected for viewing on the map or plot.
+Includes Super Regions and Sub Regions.
+
+- All regions (Super Regions and Sub Regions) **MUST** have a unique numeric region
+  identifier.
+    - Regions and Region Collections **MUST NOT** be identified by any of the following
+      reserved strings:
+        - `undefined`
+        - `regions`
+        - `collections`
+        - `metadata`
+
```

Comment:
Regions are also called land subdivisions on the computer side.

Reply:
Is there a change you're suggesting to the list of forbidden identifiers here?

Reply:
no change. just put somewhere in the doc that regions are called land subdivisions in the supercomputer side

Reply:
Does this cover that need? https://github.com/nsidc/snow-today-webapp-server/pull/46/files#diff-d976d0db94b08b3494a79581a123a43e65f068687e88ae4dd7d898ed4f99882cR77